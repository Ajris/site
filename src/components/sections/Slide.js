import React from "react";
import {Carousel} from "react-bootstrap";
import '../../App.css';
import Image from "react-bootstrap/Image";

export default class Slide extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            name:props.name,
            image:props.image,
        };
    }

    render() {
        return (
            <div>
                <Image className="d-block w-100 img-fluid slide-img" src={this.state.image} alt={"0"} fluid/>
                <Carousel.Caption>
                    <h3>{this.state.name}</h3>
                </Carousel.Caption>
            </div>
        )
    }
}