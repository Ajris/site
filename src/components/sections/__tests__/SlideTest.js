import {create} from "react-test-renderer";
import Slide from "../Slide";
import React from "react";

describe("Slide component", () => {
    test("it matches the snapshot", () => {
        const component = create(<Slide/>);
        expect(component.toJSON()).toMatchSnapshot();
    });
});