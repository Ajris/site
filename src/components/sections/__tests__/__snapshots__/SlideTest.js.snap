// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide component it matches the snapshot 1`] = `
<div>
  <img
    alt="0"
    className="d-block w-100 img-fluid slide-img img-fluid"
  />
  <div
    className="carousel-caption"
  >
    <h3 />
  </div>
</div>
`;
